Danki2/Assets/Scripts/Stats/StatsManager.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class StatsManager
+{
+    private readonly Stats baseStats;
+    private readonly Stats frameStats;
+
+    public StatsManager(Stats baseStats)
+    {
+        this.baseStats = baseStats;
```

Comment:
I think we're going with _ prefix for private field names rather than using 'this.' since it's the unity standard.

Reply:
Fair enough, happy to go for this convention